models/base.rb

```diff
@@ -76,7 +76,8 @@ def inspect
     def as_json(options = {})
       hash = {}
       self.class.json_fields.each{ |field_name|
-        next if !respond_to?(field_name) ||
+        next if options[:except]&.include?(field_name.to_sym) ||
```

Comment:
Can't we remove the `.to_s` part from `json_fields << name.to_s` so that we don't have to do `field_name.to_sym`? Similarly for `vrp_result_fields`?